e2e/utils/runtime-utils.ts

```diff
@@ -20,7 +20,7 @@ export async function initializeRuntimeIfModalPresented(page: Page): Promise<voi
 }
 
 async function isSecuritySuspended(page: Page): Promise<boolean> {
-  return exists(page, '//*[@data-test-id="security-suspended-msg"]');
+  return exists(page, '//*[@data-test-id="security-suspended-msg"]', { timeout: 5000 });
```

Comment:
Increase timeout to 5 seconds is needed. The `goto()` triggers page loading which takes few seconds so the default 1 second wait in `exists()` is not enough.